src/tracing/json/json_tracing.cpp

```diff
@@ -295,6 +295,17 @@ void JsonBackend::TraceCounter(const char * label)
     OutputValue(value);
 }
 
+void JsonBackend::TraceMetric(const char * label, int val)
+{
+    ::Json::Value value;
+    value["label"]       = label;
+    value["label_value"] = val;
```

Comment:
maybe just "value" ? since this is an object, it seems obvious the value is associated with the label